formats/polus-feather-to-tabular-plugin/tests/plugin_test.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import sys
+from pathlib import Path
+import filepattern
+from src import main
+
+class Convert:
+        
+    def filePattern():
+        filePattern = {
+        'csv' : '.*.csv',
+        'parquet' : '.*.parquet'
+        }
+        return filePattern
+    
+        
+class PluginTest(unittest.TestCase):
+    """ Tests to ensure the plugin is operating correctly """
+    def setUp(self):
+        self.convert = Convert()
+    
+    
+    inpDir = 'formats/polus-feather-to-tabular-plugin/tests/data/input'
+    outDir = 'formats/polus-feather-to-tabular-plugin/tests/data/output'
```

Comment:
If we have vaex installed, can we just download a couple of example files to use in testing? Otherwise, these tests are going to fail if we don't do some grunt work beforehand.

Reply:
I can access hdf5 files and sample dataframes directly form vaex but have not been able to find any sample feather files via vaex